lib/round.rb

```diff
@@ -0,0 +1,56 @@
+# require './lib/turn'
+
+class Round
+  attr_reader :turns, :number_correct
+  attr_accessor :deck
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+  end
+
+  def current_card()
+    return @deck.cards[0]
+  end
+
+  def take_turn(guess)
+    # require 'pry'; binding.pry
+    this_turn = Turn.new(guess, current_card())
```

Comment:
If you don't need to pass a method any arguments, you can leave off the () -- If you've had any experience in other programming languages prior to this, you might have noticed that it's common practice to put the () after method declarations, but in ruby, we don't need to do that if there aren't any arguments that we need to pass.